hack/chains/cosign.key

```diff
@@ -0,0 +1,11 @@
+-----BEGIN ENCRYPTED COSIGN PRIVATE KEY-----
```

Comment:
I know this is for testing, but I'd feel a lot better if we didn't commit any private keys. Can we generate this on demand?

Reply:
The need for a shared private key is so that the golden images and their attestations have a known key as signature. I've been trying to create images that we can use in https://quay.io/organization/hacbs-contract-demo/ that we could point ec to. I'd like for those to be usable and re-buildable (via `hack/rebuild.sh`) over time. For self-contained workflows like (acceptance) tests, we could generate keys, but I don't think it's worth it

Reply:
Can we store the private key as a secret here in GitHub?

Reply:
Hmm, we need to provide it to Chains running the HACBS stack, currently only the a dev cluster someone would spin up on their own is a viable option. With some investment we could run a near-HACBS stack in the development environment on Kind and then have it run on schedule in GitHub Actions to rebuild the golden images, though we're some ways off from that...